docs/whatsnew/v0-6-2.rst

```diff
@@ -4,6 +4,9 @@ v0.6.2
 API changes
 ###########
 
+* The `Flow` parameter now raises a `FutureWarning`, as it is to be removed
+  in a following version.
```

Comment:
```suggestion
* The `Flow` parameter `bidirectional` now raises a `FutureWarning`,
  as it is to be removed in a following version.
```

Reply:
TYSM for fixing this! That would have been confusing otherwise...